frontend/src/app/testQueue/TestCardForm/TestCardForm.utils.tsx

```diff
@@ -228,45 +229,141 @@ export const convertFromMultiplexResponse = (
   });
 };
 
-export const areAOEAnswersComplete = (
+export const AoeValidationErrorMessages = {
+  COMPLETE: "COMPLETE",
+  SYMPTOM_VALIDATION_ERROR: "SYMPTOM_VALIDATION_ERROR",
+  INCOMPLETE: "INCOMPLETE",
+  UNKNOWN: "UNKNOWN",
+} as const;
+
+const REQUIRED_AOE_QUESTIONS_BY_DISEASE: {
+  [key in AOEFormOption]: Array<keyof AoeQuestionResponses>;
```

Comment:
Opted here not to do `[key in AOEFormOption]?` and instead require all `AOEFormOption` values here need a corresponding entry so that when new disease forms are added, the frontend won't compile unless the new disease is mapped to required AOE questions.